pybamm/models/full_battery_models/lithium_ion/base_lithium_ion_model.py

```diff
@@ -36,14 +36,10 @@ def set_standard_output_variables(self):
             }
         )
 
-    def set_reactions(self):
-
-        # Should probably refactor as this is a bit clunky at the moment
-        # Maybe each reaction as a Reaction class so we can just list names of classes
-        icd = " interfacial current density"
-        self.reactions = {
-            "main": {
-                "Negative": {"s": 1, "aj": "Negative electrode" + icd},
-                "Positive": {"s": 1, "aj": "Positive electrode" + icd},
-            }
-        }
+    def set_other_reaction_submodels_to_zero(self):
```

Comment:
instead of having `set_other_reaction_submodels_to_zero` could we have something like `set_reaction_submodels(self,[list of reactions]):` which then sets the appropriate models from the list (and sets `NoReaction` for things that aren't in the list)? I'm just thinking of how this extends in the future and works with model options. It would be nice to have an option "reactions" which takes a list of names of any reactions you want to include. Actually it looks like this already happens, and these are always zero for li-ion. Maybe there could be a comment or this method renamed, as seeing it in e.g. DFN was a little confusing -- it wasn't clear which reactions this referred to.